com.supermodel3.Supermodel.yaml

```diff
@@ -0,0 +1,47 @@
+id: com.supermodel3.Supermodel
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: supermodel
+separate-locales: false
+finish-args:
+  - --device=all
+  - --persist=.supermodel
+  - --filesystem=home
+  - --filesystem=host:ro
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --allow=bluetooth
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: supermodel
+    buildsystem: simple
+    build-commands:
+      - make
+          -f Makefiles/Makefile.UNIX
+          NET_BOARD=1
+      - install -Dm 0755 bin/supermodel ${FLATPAK_DEST}/bin/supermodel
+      - install -Dm 0644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 0644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm 0644 supermodel128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - mkdir -p ${XDG_CONFIG_HOME}/.supermodel/NVRAM
```

Comment:
this is not valid. This is done during the build of the package. So it will never leave the tmpfs of the build sandbox.

Again this requirements is also broken for any packaging.

Reply:
Not sure what you mean.  Which line?

Reply:
31 and below. Everything that copies to the home dir.